pkgs/development/compilers/swift/default.nix

```diff
@@ -232,23 +244,21 @@ stdenv.mkDerivation rec {
     # https://bugs.swift.org/browse/SR-4633
     patch -p1 -d swift -i ${./patches/icu59.patch}
 
+    # https://bugs.swift.org/browse/SR-5779
+    sed -i -e 's|"-latomic"|"-Wl,-rpath,${clang.cc.gcc.lib}/lib" "-L${clang.cc.gcc.lib}/lib" "-latomic"|' swift/cmake/modules/AddSwift.cmake
```

Comment:
Do you know if this can be made to work with libc++ instead of gcc-based (libstdc++-based) clang? Just curious.

Reply:
`libatomic.so` is required for `__atomic_load` and other `__atomic_*` functions. If `libc++` provides them, we may eliminate dependency to GCC apckage.